Cars/CarsServices/Controllers/AdminPanelController.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using CarsData.Models;
+using CarsRepository.Interface;
+//using Cars.Data.Repository;
+using CarsServices.ViewModels;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CarsServices.Controllers
+{
+    public class AdminPanelController : Controller
+    {
+        ICarRepository _Car;
+        
+        public AdminPanelController(ICarRepository iCar)
+        {
+            _Car = iCar;
+        }
+
+        [HttpGet]
+        public ViewResult Index()
+        {           
+            return View(_Car.GetAllCars());
+        }
+
+        public ViewResult Edit(int CarId)
+        {
+            Car car = _Car.GetAllCars()
```

Comment:
First of all, you have a method to get only one car by ID. Second one is you have to create s service if you want to process your data from DB. 
In a Controller you can only validate your data, not process it.